### DISCUSSION TITLE
Querying and Setting HwcTimer via HA

### DISCUSSION THREAD

**Discussion by henkela** [john30/ebusd#634] [Q&A]
Hi, 
I'm fairly new to Home Assistant and ebsud but the ebusd-Integration with mqtt has been working great so far. I can see the temperatures and status of my Vaillant multiMatic 700 and vr 70. However, I would like to have some Input fields on a Dashboard to set the Heating times for HwcTimer.* for example. Unfortunately, I can't even get the current fields. I've tried the following config in the configuration.yml: 

mqtt:
  sensor:
    - name: "Hwc Timer Monday"
      state_topic: "ebusd/700/hwctimer.monday/get"

I also tried it without the '/get' part. The value is just reported as unknown. Do you have a hint what I'm missing? 

Best,
Andreas

**Comment by john30** [comment]
which integration is that exactly? the [ebusd builtin MQTT discovery](https://github.com/john30/ebusd/wiki/MQTT-integration#home-assistant) or something else?
